samples/nvidia_car_classification/module.yml

```diff
@@ -7,6 +7,21 @@ parameters:
     module: samples.nvidia_car_classification.overlay
     # specify the drawfunc's python class from the module
     class_name: Overlay
+    rendered_objects:
+      Primary_Detector:
+        Car:
+          bbox:
+            color: 00FF00FF  # Green
+            thickness: 2
+          label:
+            color: FFFFFFFF  # White
+            font_scale: 0.5
+            thickness: 1
+            format: ['{label} #{track_id}']
```

Comment:
please, add comment that format is overriden by a function which adds class labels detected.

Reply:
Added.